mocha-runner.html

```diff
@@ -2,14 +2,14 @@
 <head>
   <meta charset="utf-8">
   <title>Mocha Tests</title>
-  <link href="https://cdn.rawgit.com/mochajs/mocha/2.2.5/mocha.css" rel="stylesheet" />
+  <link href="mocha/mocha.css" rel="stylesheet" />
 </head>
 <body>
   <div id="mocha"></div>
 
   <script src="https://cdn.rawgit.com/jquery/jquery/2.1.4/dist/jquery.min.js"></script>
-  <script src="https://cdn.rawgit.com/Automattic/expect.js/0.3.1/index.js"></script>
-  <script src="https://cdn.rawgit.com/mochajs/mocha/2.2.5/mocha.js"></script>
+  <script src="expect.js/index.js"></script>
+  <script src="mocha/mocha.js"></script>
```

Comment:
I noticed on the flight that the mocha server fails without internet connection. This is why that demo on thursday failed.

this fixes it by having the mocha server host the node modules needed. jQuery is ignored because it's probably not necessary although i'm not 100% sure yet.